modules/other-integrations/pages/execute-dbt-teradata-transformations-in-airflow-with-cosmos.adoc

```diff
@@ -0,0 +1,248 @@
+In this example, we will explain how to use astronomer cosmos to run dbt teradata transformations in airflow against Teradata vantage database.
+
+== Prerequisites
+* Access to a Teradata Vantage instance, version 17.10 or higher.
++
+include::ROOT:partial$vantage_clearscape_analytics.adoc[]
+* Python 3.8, 3.9, 3.10 or 3.11 installed.
+* python3-env, python3-pip, pipx installed.
+[source, bash]
+sudo apt install -y python3-venv python3-pip pipx
+
+NOTE: Use `Windows PowerShell` command-line shell on `Windows` OS to try this quickstart example.
+
+== Install Apache Airflow
+1. Set the AIRFLOW_HOME environment variable. Airflow requires a home directory and uses ~/airflow by default, but you can set a different location if you prefer. The AIRFLOW_HOME environment variable is used to inform Airflow of the desired location.
++
+[source, bash]
+----
+export AIRFLOW_HOME=~/airflow
+----
+2. Install `apache-airflow` stable version 2.9.2 from PyPI repository.:
++
+[tabs]
+====
+Windows::
++
+--
+[source, bash]
+----
+AIRFLOW_VERSION=2.9.2
+PYTHON_VERSION="$(python --version | cut -d " " -f 2 | cut -d "." -f 1-2)"
+CONSTRAINT_URL="https://raw.githubusercontent.com/apache/airflow/constraints-${AIRFLOW_VERSION}/constraints-${PYTHON_VERSION}.txt"
+pip install "apache-airflow==${AIRFLOW_VERSION}" --constraint "${CONSTRAINT_URL}"
+----
+--
+MacOS::
++
+--
+[source, bash]
+----
+AIRFLOW_VERSION=2.9.2
+PYTHON_VERSION="$(python3 --version | cut -d " " -f 2 | cut -d "." -f 1-2)"
+CONSTRAINT_URL="https://raw.githubusercontent.com/apache/airflow/constraints-${AIRFLOW_VERSION}/constraints-${PYTHON_VERSION}.txt"
+pip install "apache-airflow==${AIRFLOW_VERSION}" --constraint "${CONSTRAINT_URL}"
+----
+--
+Linux::
++
+--
+[source, bash]
+----
+AIRFLOW_VERSION=2.9.2
+PYTHON_VERSION="$(python3 --version | cut -d " " -f 2 | cut -d "." -f 1-2)"
+CONSTRAINT_URL="https://raw.githubusercontent.com/apache/airflow/constraints-${AIRFLOW_VERSION}/constraints-${PYTHON_VERSION}.txt"
+pip install "apache-airflow==${AIRFLOW_VERSION}" --constraint "${CONSTRAINT_URL}"
+----
+--
+====
+
+3. Install the Apache Airflow Teradata provider stable version from the corresponding PyPI repository.
++
+[source, bash]
+----
+pip install "apache-airflow-providers-teradata"
+----
+
+== Install Astronomer Cosmos
+[source, bash]
+----
+pip install "astronomer-cosmos"
+----
+
+== Install dbt
+1. Create a new python environment to manage dbt and its dependencies. Activate the environment:
++
+[tabs]
```

Comment:
Again same as above, we might not need the tabs since all is the same.

Reply:
Removed.